tv_apps/smart-home/manifest.webapp

```diff
@@ -30,6 +30,12 @@
       }
     }
   },
+  "connections": {
+    "appdeck-channel": {
+      "description": "communicate channel for app-deck",
```

Comment:
I'd suggest more specific way to describe, like `for app-deck to add/remove application cards`